aws/stage.js

```diff
@@ -123,7 +123,7 @@ const stage = (dir, repo, branch) => {
   sh.echo('...monkey patching html paths')
   indexFiles.forEach((file) => {
     cache[file] = sh.cat(file)
-    sh.sed('-i', pathRegExp, (group, g1, g2) => `${g1}/${stagedPath}/${g2}`, file)
+    sh.sed('-i', pathRegExp, (group, g1, g2) => `${g1}/${stagedPath}${g2 !== '"' ? `/${g2}` : g2}`, file)
```

Comment:
Yep, regex matches as below. We insert the staging path in between the groups. Staging path is always abs, so the starting slash is disregarded.

```
href="/"

group1: href="
group2: "

result: href="staging/path"
```
>No trailing slash after staging path ^

Before, we ALWAYS prepended group2 with a slash.  So, the closing `"` was always `/"`.  Now, we only do that if group two does not end with a `"`:

```
href="/nested/path"

group1: href="
group2 nested/path"

result: href="staging/path/nested/path"
```

Notice `staging/path` has a slash after it, so group2 is separated.  Only because group2 was not a closing quote.  Otherwise, we'd get: `staging/pathnested/path`.